tests/e2e/helpers/database.ts

```diff
@@ -57,32 +58,34 @@ export async function discoverSentinelDatabase(databaseParameters: SentinelParam
  * @param databaseParameters The database parameters
  */
 export async function addNewREClusterDatabase(databaseParameters: AddNewDatabaseParameters): Promise<void> {
-    //Fill the add database form
+    // Fill the add database form
     await addRedisDatabasePage.addAutodiscoverREClucterDatabase(databaseParameters);
-    //Click on submit button
-    await t.click(addRedisDatabasePage.addRedisDatabaseButton);
-    //Wait for database to be exist in the list of Autodiscover databases and select it
-    await t.expect(autoDiscoverREDatabases.databaseNames.withExactText(databaseParameters.databaseName ?? '').exists).ok('The existence of the database', { timeout: 10000 });
-    await t.typeText(autoDiscoverREDatabases.search, databaseParameters.databaseName ?? '');
-    await t.click(autoDiscoverREDatabases.databaseCheckbox);
-    //Click Add selected databases button
-    await t.click(autoDiscoverREDatabases.addSelectedDatabases);
-    await t.click(autoDiscoverREDatabases.viewDatabasesButton);
+    // Click on submit button
+    await t
+        .click(addRedisDatabasePage.addRedisDatabaseButton)
+    // Wait for database to be exist in the list of Autodiscover databases and select it
+        .expect(autoDiscoverREDatabases.databaseNames.withExactText(databaseParameters.databaseName ?? '').exists).ok('The database not displayed', { timeout: 10000 })
+        .typeText(autoDiscoverREDatabases.search, databaseParameters.databaseName ?? '')
+        .click(autoDiscoverREDatabases.databaseCheckbox)
+    // Click Add selected databases button
+        .click(autoDiscoverREDatabases.addSelectedDatabases)
+        .click(autoDiscoverREDatabases.viewDatabasesButton);
```

Comment:
I suggest to split the `await t` to avoid issues

Reply:
Why it is better to split? and what possible issues do you mean?
 to split only when there are comments between actions, like that?
 `    await t.click(addRedisDatabasePage.addRedisDatabaseButton);
    // Wait for database to be exist in the list of Autodiscover databases and select it
        await t
        .expect(autoDiscoverREDatabases.databaseNames.withExactText(databaseParameters.databaseName ?? '').exists).ok('The database not displayed', { timeout: 10000 })
        .typeText(autoDiscoverREDatabases.search, databaseParameters.databaseName ?? '')
        .click(autoDiscoverREDatabases.databaseCheckbox);
    // Click Add selected databases button
        await t
        .click(autoDiscoverREDatabases.addSelectedDatabases)
        .click(autoDiscoverREDatabases.viewDatabasesButton);`